packages/webchat/src/main.tsx

```diff
@@ -103,9 +91,10 @@ class Web extends React.Component<MainProps> {
       this.postMessageToParent('setWidth', this.config.containerWidth)
     }
 
-    if (this.config.reference) {
-      await this.props.setReference!()
-    }
+    // TODO: properly implement of deprecate this feature
+    //if (this.config.reference) {
+    //  await this.props.setReference!()
+    //}
```

Comment:
I'm pretty sure this feature just stops existing

Reply:
Well, I think so too but since I'm not 100% sure, I preferred simply commenting it out.